tests/Feature/ProjectControllerTest.php

```diff
@@ -0,0 +1,429 @@
+<?php
+
+namespace Tests\Feature;
+
+use Tests\TestCase;
+use App\Project;
+use Faker\Factory;
+use Illuminate\Foundation\Testing\WithFaker;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\WithoutMiddleware;
+
+class ProjectControllerTest extends TestCase
+{
+    use DatabaseMigrations;
+    use WithoutMiddleware;
+    /**
+     * A basic test example.
+     *
+     * @return void
+     */
+    public function testExample()
+    {
+        $this->assertTrue(true);
+    }
+
+    public function testListProjectSuccess()
+    {
+        $project = factory(Project::class)->create();
+        $response = $this->get('project');
+        $response->assertStatus(200);
+    }
+
+    public function testDeleteProjectSuccess()
+    {
+        $array = Factory(Project::class)->create()->toArray();
+        $response = $this->call('DELETE', 'project/destroy', $array);
+        $this->assertEquals(200, $response->status());
+        $this->assertDatabaseMissing('projects', [
+        'name' => $array['name'],
+        'information' => $array['information'],
+        'deadline' => $array['deadline'],
+        'type' => $array['type'],
+        'status' => $array['status']
+        ]);
+    }
+
+    public function testEditProjectSuccess()
+    {
+        $json = '{"id":1,"name":"test","information":"tr",'.
+        '"deadline":"2018-02-25","type":"single","status":"planned"}';
+        $array1 = [
+        'id' => 1,
+        'name' => 'test',
+        'information' => 'tr',
+        'deadline' => '2018-02-25',
+        'type' => 'single',
+        'status' => 'planned',
+        ];
+        $array = Factory(Project::class)->create()->toArray();
+        $response = $this->json('PUT', 'project/update', $array1);
+        $this->assertSame($json, $response->getContent());
+        $response->assertStatus(200, $response->status());
+        $response->assertSuccessful();
+    }
+
+    public function testAddProjecSuccess()
+    {
+        $json = '{"name":"testapp","information":"traniing",'.
```

Comment:
Thanks for verifying the response here. You should do this in all test cases.

Reply:
yep sir